### PR TITLE
Cleanup services module

### PR DESCRIPTION
**Pull request by jridgewell**
- Don’t call `getServices` with a second parameter (it’s not expecting it)
- Reorder conditionals to make code clearer

### CODE REVIEW

**Comment by ampsauce** [comment]
**Tests failed.** Automated cross-browser testing via Sauce Labs and Travis CI shows that the JavaScript changes in this pull request are: **BUSTED**

Commit: 561d569e6f5839b3188ef3fef355ffd0a519b22e
Build details: https://travis-ci.org/ampsauce/amphtml/builds/99502591

(*Please note that this is a [fully automated](https://github.com/twbs/savage) comment.*)

File: src/service.js [hunk 1]
```diff
@@ -78,18 +80,15 @@ export function getElementService(win, id, providedByElement) {
...
-    if (s.promise) {
-      return s.promise;
-    }
-    // If meanwhile an object was created, we make a promise from it
+    // If a service was registered with getService, we make a promise from it
     // which we will return in future invocations.
-    if (s.obj) {
-      return s.promise = Promise.resolve(s.obj);
+    if (!s.promise) {
+      s.promise = Promise.resolve(s.obj);
```
Comment (cramforce) [inline src/service.js]: Web S=should probably have an assert here that s.obj is actually an object.

**Comment by cramforce** [comment]
LGTM

**Comment by cramforce** [comment]
(aside: Please add "NEEDS REVIEW" label when sending PR).

**Pull request merged**